docs/answers.md

```diff
@@ -38,6 +38,58 @@ const annotateData = (array) => array.reduce(reducer, [])
 const newData = annotateData(data)
 ```
 
+## Objects and Classes
+
+### Delays
+
+```{js}
+class Delay {
+  constructor(initialValue){
+    this.nextValue = initialValue
+  }
+
+  call(nextValue) {
+    let previousValue = this.nextValue
+    this.nextValue = nextValue
+    return previousValue
+  }
+}
+```
+
+### Filtering
+
+```{js}
+class Filter {
+  constructor(...values){
+    this.filterValues = values
+  }
+
+  call(inputValue){
+    return this.filterValues.some((value) => value === inputValue) ? null : inputValue
+  }
+}
+```
+
+### Pipelines
+
+```{js}
+class Pipeline {
+    constructor(...pipes){
+        this.pipes = pipes
+    }
+
+    call(inputValue){
+        let returnValue
+        for (let pipe of this.pipes){
+            returnValue = (() => pipe.call(inputValue))()
```

Comment:
Why introduce the anonymous function call? Why not simply `value = pipe.call(value)`? (Assuming you unify the two variable `inputValue` and `returnValue` as described above.)